NewApp/ngsa-csharp/Ngsa.DataService/Dockerfile

```diff
@@ -0,0 +1,45 @@
+### Build and Test the App
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS test
+
+SHELL ["/bin/bash", "-c"]
+
+# dotnet compiler options
+ARG TAG=false
+
+EXPOSE 4120
+
+### copy the source and tests
+COPY NewApp/ngsa-csharp /src
+
+WORKDIR /src/Ngsa.DataService
+
+# build the app
+RUN if [ "$TAG" == "true" ]; then dotnet publish -c Release -o /app --version-suffix ''; else dotnet publish -c Release -o /app; fi
+
+WORKDIR /src/Ngsa.DataService.Tests
+ENTRYPOINT [ "./runtests" ]
+
+###########################################################
+
+
+### Build the runtime container
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine AS release
+
+### if port is changed, also update value in Constants.cs
+EXPOSE 4122
```

Comment:
why is the release container changed to 4122?

Reply:
The data service runs on 4122. The app still runs on 4120.